packages/react-query/src/QueryDefaultOptionsProvider.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+import * as React from 'react'
```

Comment:
```suggestion
import { createContext, useContext } from 'react';
import type { ReactNode, FC } from 'react';
```

Doing it this way makes the code clearer and helps reduce the bundle size by not importing unnecessary things. What do you think about this approach?